app/controllers/organizations_controller.rb

```diff
@@ -43,4 +41,23 @@ def events
       PaginationPresenter
     )
   end
+
+  def evidence_items
+    evidence_items = EvidenceItem.order('evidence_items.id DESC')
+      .eager_load(:submitter)
+      .where('users.id': user_ids)
```

Comment:
Does this work as written? I'd think you'd need `'users.id' => user_ids` as `thing: other_thing` is shorthand for `:thing => other_thing` and `users.id` isn't a symbol, its a string.